sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md

```diff
@@ -48,68 +55,44 @@ We need to ensure that this [environment checklist][ready-to-run-checklist] is c
 </dependencies>
 ```
 
-### Register your Web API
-You can follow *azure-spring-boot-sample-active-directory-resource-server* sample to add `ResourceAccessGraph.read`, `ResourceAccessGraphCustomResources.read` scopes. 
-Convention current application id url is `api://sample-client-id`, ; the application id url of the sample *azure-spring-boot-sample-active-directory-resource-server* is `custom-client-id`.  
-After adding as shown below:
-
-   ![API Permissions](resource/resource-server-obo-add-scope.png)
-
-### Add API permissions
-The current Web API will access Graph API and Custom API. 
-
-Sign in to the [Azure portal][azure-portal]. If you have access to multiple tenants, use the **Directory + subscription** filter  in the top menu to select the tenant containing your client app's registration.
-
-#### Add Graph API Permission
-1. Select **Azure Active Directory** > **App registrations**, and then select your current sample application (not your web API).
```

Comment:
We don't need the Graph API permissions now?

Reply:
I'm using photos instead of text description.